package.json

```diff
@@ -104,7 +110,7 @@
     "viewsWelcome": [
       {
         "view": "openstax-controls",
-        "contents": "[Open ToC Editor](command:openstax.showTocEditor)\n[Open Image Upload](command:openstax.showImageManager)\n[Push Content](command:openstax.pushContent)"
+        "contents": "[Open ToC Editor](command:openstax.showTocEditor)\n[Open Image Upload](command:openstax.showImageUpload)\n[Push Content](command:openstax.pushContent)\n[Tag Content](command:openstax.tagContent)"
```

Comment:
I think something went wrong in a merge along the way as this should stay `showImageManager` (the button is otherwise broken on this branch).